init.sh

```diff
@@ -13,4 +13,6 @@ if [ ! -L $FCCSWBASEDIR/.git/hooks/pre-commit ]; then
 fi
 
 weekday=`date +%a`
+
 source /cvmfs/fcc.cern.ch/sw/views/releases/externals/94.0.0/x86_64-slc6-gcc62-opt/setup.sh
+export CMAKE_PREFIX_PATH=/afs/cern.ch/work/v/vavolkl/public/fcc.cern.ch/sw/dev/podio:$CMAKE_PREFIX_PATH
```

Comment:
This line should be removed.

Reply:
yes. can you remind me when we have externals with the new podio available? Then I'll change it. Thanks!

Reply:
Newest `podio` (current master) can be tested with today's nightlies:

```
/cvmfs/fcc.cern.ch/sw/views/nightlies/externals/latest/x86_64-slc6-gcc62-opt/setup.sh
```

We can modify the `init.sh` file to point to that `setup.sh` and if it works, we can consider a new release of the externals with a new `podio` version and other packages if needed.